src/store/segmenter/segmenterSlice.ts

```diff
@@ -131,7 +138,7 @@ export const segmenterSlice = createSlice({
       action: PayloadAction<{
         inputShape: Shape;
         modelSelection: SegmenterModelProps;
-        model: LayersModel;
+        model: LayersModel | GraphModel;
```

Comment:
We should probably just create our own `TFModel` type and use that, since we duplicate this here, in `segmenterCompiledModelSelector`, `segmenterFittedModelSelector`, etc